v0/destinations/hs/HSTransform-v1.js

```diff
@@ -57,6 +57,7 @@ const processLegacyIdentify = async (message, destination, propertyMap) => {
   // rETL source
   let endpoint;
   const response = defaultRequestConfig();
+  response.method = defaultPostRequestConfig.requestMethod;
```

Comment:
Let's move this to its respective section rather than defaulting to `POST` at the top for all the calls.

Reply:
this was present in the final statement but it would change for all objects update as it is patch. then for all calls this will be post. this is as the default behaviour.

Reply:
We are overriding this value for rETL source below. IMO putting it in the respective section i.e after line 114 would make it more readable and won't conflict with another endpoint if that is going to add later.